docs: Adding a config section for the helper (#272)
awslabs/amazon-ecr-credential-helper: README.md

The credentials must have a policy applied that
[allows access to Amazon ECR](http://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr_managed_policies.html).

### Amazon ECR Docker Credential Helper 

> | Environment Variable  | Default Value | Description                                                       |

Reviewer: The default values are empty for the first one and OS-dependent for the second, so it's a bit hard to describe them as "Default Value" here.

```suggestion
| Environment Variable  | Sample Value | Description                                                       |
```

Author: Chagned and rebased

https://github.com/awslabs/amazon-ecr-credential-helper/blob/efd1603705e91c31bf28cbbc5dba8c3e09c63100/ecr-login/config/cache_dir.go I thought though that we always return `"~/.ecr"` regardless of OS?  It would be nice to document the default value if possible but it's not a big deal if not

Reviewer: My apologies; I thought we had already merged https://github.com/awslabs/amazon-ecr-credential-helper/pull/250 which changes the behavior.

Author: Ah, OK cool, I'll leave it as it is

| --------------------- | ------------- | ----------------------------------------------------------------- |

> | AWS_ECR_DISABLE_CACHE | 1             | Disables the local file auth cache if set to a zero length string |

Reviewer: The description here is a bit incorrect:

```suggestion
| AWS_ECR_DISABLE_CACHE |               | Disables the local file auth cache if set to a non-empty value |
```

> | AWS_ECR_CACHE_DIR     | ~/.ecr        | Customises the local file auth cache directory location           |

Reviewer: nit: We're using American English spelling in this repository, but we can avoid the differences by picking a different word.

```suggestion
| AWS_ECR_CACHE_DIR     | ~/.ecr        | Specifies the local file auth cache directory location           |
```
